packages/subapp-web/lib/start.js

```diff
@@ -63,6 +64,7 @@ module.exports = function setup() {
 
       // default group _ subapps should all run independently
       if (xarcSubappSSR._) {
+        startTimes[`load-subapps`] = Date.now();
```

Comment:
if `startTimes` is only used within this closure, then it can simply be a variable.